src/pitoco/core.clj

```diff
@@ -400,8 +400,11 @@
     ;; Run `tcptrace` to generate the dat files.
     (-> (conj '[sh -c] (str "tcptrace -el " (.getAbsolutePath (io/file pcap-path))))
         (sh/process {:out :string :err :string :dir folder-path})
-        :out
         deref
+        (doto (as-> % (when (= 127 (:exit %))
```

Comment:
This should throw a nice error like this:
```
Execution error (ExceptionInfo) at pitoco.core/dat-files-from-pcap-path (core.clj:405).
sh: line 1: tcptrace: command not found
```

The old behavior was: return an empty list, that is confusing.